bg_atlasgen/validate_atlases.py

```diff
@@ -152,18 +175,20 @@ def validate_atlas(atlas_name, version, validation_functions):
     if not updated:
         update_atlas(atlas_name)
 
-    # list to store the errors of the failed validations
-    failed_validations = {atlas_name: []}
-    successful_validations = {atlas_name: []}
+    validation_results = {atlas_name: []}
 
     for i, validation_function in enumerate(validation_functions):
         try:
             validation_function(BrainGlobeAtlas(atlas_name))
-            successful_validations[atlas_name].append(validation_function)
+            validation_results[atlas_name].append(
+                (validation_function.__name__, None, str("Pass"))
+            )
         except AssertionError as error:
-            failed_validations[atlas_name].append((validation_function, error))
+            validation_results[atlas_name].append(
+                (validation_function.__name__, str(error), str("Fail"))
+            )
 
-    return successful_validations, failed_validations
+    return validation_results
```

Comment:
Think we should to rebase or merge `main` here, as there's some changes here that are the same as those already in current `main`, ie commit hash: 03392f3 (but with a different commit hash: cfccbb8).

These changes are unrelated to the current PR as far as I can tell?

Reply:
I see what you mean but I'm gonna need your help with this

Reply:
@alessandrofelder, can we do this together whenever you have the time?